setup.py

```diff
@@ -27,7 +27,7 @@
                     'knimin.handlers': []},
       extras_require={'test': ["nose >= 0.10.1", "pep8", "flake8", "mock",
                                "requests-mock"]},
-      install_requires=['psycopg2', 'tornado==3.1.1', 'WTForms==2.0.1',
+      install_requires=['psycopg2', 'tornado>=4.0.0', 'WTForms==2.0.1',
```

Comment:
is >= 4.0.0 accurate? i thought testing was happening right now with 4.2.2 or something?